testinfra/modules/package.py

```diff
@@ -146,3 +149,20 @@ def version(self):
     def release(self):
         return self.check_output('rpm -q --queryformat="%%{RELEASE}" %s',
                                  self.name)
+
+
+class AlpinePackage(Package):
+
+    @property
+    def is_installed(self):
+        return self.check_output("apk -e info %s", self.name) == self.name
+
+    @property
+    def version(self):
+        out = self.check_output("apk -e -v info %s", self.name).split("-")
+        return out[-2]
+
+    @property
+    def release(self):
+        out = self.check_output("apk -e -v info %s", self.name).split("-")
```

Comment:
`out = out[len(name)+1:].split('-', 1)[1]`

Reply:
release do not agree, they will work with packages containing "-" because because the list is addressed from the end, 

the output of the command contains the structure
name-version-release
and the first from the end out[-1] it is release